src/glue.rs

```diff
@@ -32,4 +32,67 @@ impl Glue {
             }
         }
     }
+
+    pub fn select_as_string(&mut self, query: &Query) -> Result<Vec<Vec<String>>> {
+        // TODO: Make this more efficient by converting earlier
+        match self.execute(query) {
+            Ok(Payload::Select { labels, rows }) => Ok(vec![labels] // Gross
+                .into_iter()
+                .chain(
+                    rows.into_iter()
+                        .map(|row| {
+                            row.0
+                                .into_iter()
+                                .map(|value| value.into())
+                                .collect::<Vec<String>>()
+                        })
+                        .collect::<Vec<Vec<String>>>(),
+                )
+                .collect()),
+            _ => Err(ExecuteError::QueryNotSupported.into()),
+        }
+    }
+}
+
+#[cfg(feature = "sled-storage")]
+#[cfg(test)]
+mod tests {
+    use crate::{parse_sql::parse, Glue, Payload, Row, SledStorage, Value};
+    #[test]
+    fn eq() {
+        let path = "test";
+        std::fs::remove_dir_all(&path);
+        let sled = SledStorage::new(path).unwrap();
```

Comment:
Wow this looks really a proper test in the perfect location, thanks!

```rust
let config = sled::Config::default()
    .path("data/using_config")
    .temporary(true);

let storage = SledStorage::try_from(config).unwrap();
```
Let's simply use `sled`s `temporary` option :)
Then we can delete two `std::fs::remove_dir_all(&path);` in here.

Reply:
Ah perfect! Thanks!